src/coprocessor/metrics.rs

```diff
@@ -98,3 +98,35 @@ lazy_static! {
             20.0, 24.0, 28.0, 32.0, 48.0, 64.0, 96.0, 128.0, 192.0, 256.0]
         ).unwrap();
 }
+
+/// `ScanCounterMetrics` is for recording range query and point query.
```

Comment:
Like raftstore, I suggest moving this to a local_metric.rs file and providing a flush function for flush to global metrics explicitly.